### PR TITLE
feat: add `kili.data_integrations`

### PR DESCRIPTION
**Pull request by Jonas1312**
doc rendering is ok

also tested:

```python
kili.data_integrations(platform="Azure", fields=["name", "platform", "status"])
```

returns:

```python
[{'name': 'Test Azure Token based ', 'platform': 'Azure', 'status': 'CONNECTED'}, {'name': 'Test Azure', 'platform': 'Azure', 'status': 'CONNECTED'}, {'name': 'Azure integration for tests', 'platform': 'Azure', 'status': 'DISCONNECTED'}, {'name': 'Test Datalake', 'platform': 'Azure', 'status': 'DISCONNECTED'}, {'name': 'Test demo-container-delegated-access', 'platform': 'Azure', 'status': 'CONNECTED'}, {'name': 'StorageClientAppSample', 'platform': 'Azure', 'status': 'DISCONNECTED'}, {'name': 'test 2', 'platform': 'Azure', 'status': 'CONNECTED'}, {'name': 'test-integration', 'platform': 'Azure', 'status': 'DISCONNECTED'}, {'name': 'test integ 1', 'platform': 'Azure', 'status': 'CHECKING'}]
```

schema:

```graphql
enum DataIntegrationStatus {
  CONNECTED
  DISCONNECTED
  CHECKING
}

enum DataIntegrationPlatform {
  AWS  Azure  GCP
}

input DataIntegrationWhere {
  status: DataIntegrationStatus
  id: ID
  name: String
  organizationId: ID
  platform: DataIntegrationPlatform
}


"""
  Retrieves page by page all dataIntegrations who verify where input parameters.
"""
dataIntegrations(where: DataIntegrationWhere!, first: PageSize!, skip: Int!): [DataIntegration!]

"""Count all dataIntegrations who verify where input parameters."""
countDataIntegrations(where: DataIntegrationWhere!): Int!
  
  
"""Get the dataIntegration's data"""
dataIntegration(where: DataIntegrationWhere!): DataIntegration
```

### CODE REVIEW

File: docs/sdk/remote_storage.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Remote storage module
```
Comment (PierreLeveau) [inline docs/sdk/remote_storage.md:1]: Cloud storage module (also change file name)

File: src/kili/queries/data_integration/__init__.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            auth: KiliAuth object
+        """
+        self.auth = auth
+
+    @typechecked
+    def data_integrations(
+        self,
+        data_integration_id: Optional[str] = None,
+        name: Optional[str] = None,
+        platform: Optional[str] = None,
```
Comment (PierreLeveau) [inline src/kili/queries/data_integration/__init__.py:36]: for platform and status, a `Literal` can be used.

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

**Review by PierreLeveau — APPROVED** [review]

File: src/kili/queries/data_integration/__init__.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        as_generator: bool = False,
+    ) -> Iterable[Dict]:
+        # pylint: disable=line-too-long
+        """Get a generator or a list of data integretations that match a set of criteria.
+
+        Args:
+            data_integration_id: ID of the data integration.
+            name: Name of the data integration.
+            platform: Platform of the data integration.
+                Possible choices: `AWS`, `Azure` or `GCP`.
```
Comment (theodu) [inline src/kili/queries/data_integration/__init__.py:54]:
```suggestion
```

File: src/kili/queries/data_integration/__init__.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            status: Status of the data integration.
+                Possible choices: `CONNECTED`, `DISCONNECTED` or `CHECKING`.
```
Comment (theodu) [inline src/kili/queries/data_integration/__init__.py:56]:
```suggestion
```

File: src/kili/queries/data_integration/__init__.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        status: Optional[Literal["CONNECTED", "DISCONNECTED", "CHECKING"]] = None,
+        organization_id: Optional[str] = None,
+    ) -> int:
+        """Count and return the number of data integrations that match a set of criteria.
+
+        Args:
+            data_integration_id: ID of the data integration.
+            name: Name of the data integration.
+            platform: Platform of the data integration.
+                Possible choices: `AWS`, `Azure` or `GCP`.
```
Comment (theodu) [inline src/kili/queries/data_integration/__init__.py:102]:
```suggestion
```

File: src/kili/queries/data_integration/__init__.py [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            status: Status of the data integration.
+                Possible choices: `CONNECTED`, `DISCONNECTED` or `CHECKING`.
```
Comment (theodu) [inline src/kili/queries/data_integration/__init__.py:104]:
```suggestion
```

**Review by theodu — APPROVED** [review]

**Pull request merged**